src/graphql/schema/collection/createCollectionRelationTailGqlFieldEntries.ts

```diff
@@ -46,11 +46,12 @@ export default function createCollectionRelationTailGqlFieldEntries <TSource, TV
             : formatName.field(`${headCollection.type.name}-by-${relation.name}`),
           {
             description: `Reads a single ${scrib.type(headCollectionGqlType)} that is related to this ${scrib.type(collectionGqlType)}.`,
+            externalFieldNameDependencies: relation._tailFieldNames, // 🔥 Needs to account for classicIds
```

Comment:
It's possible that your primary key is called "flibbledeedee" and that your "id" column is merely a foreign key.